mms/tests/unit_tests/test_model_loader.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+import os
+import shutil
+import json
+from mms.model_loader import MANIFEST_FILENAME
+from mms.model_loader import ModelLoader
+
+model_dir_path = 'my-model'
+handler_file = 'handler'
+
+manifest_invalid_data_missing_paramsFile = {"model": {}, "engine": {"engineName": "MxNet"}}
+manifest_invalid_data_missing_symbolFile = {"model": {"parametersFile": "my-model/params1"},
+                                            "engine": {"engineName": "MxNet"}}
+manifest_valid_data = {"model": {"parametersFile": "my-model/params1", "symbolFile": 'my-model/symbol.json'},
+                       "engine": {"engineName": "MxNet"}}
+
+
+def empty_file(filepath):
+
+    """
+    Utility function for creating an empty file, which will be used throughout these unit tests
+    :param filepath:
+    :return:
+    """
+
+    open(filepath, 'a').close()
```

Comment:
If we are using mocker, we probably want to use https://docs.python.org/dev/library/unittest.mock.html#mock-open to avoid messing with the filesystem.

Reply:
Discussed offline. Skipping the mock for os funcs for now.